lib/install/rust.js

```diff
@@ -0,0 +1,523 @@
+// System Objects
+var path = require('path');
+var spawn = require('child_process').spawn;
+var spawnSync = require('child_process').spawnSync;
+var Transform = require('stream').Transform;
```

Comment:
Same as above. Here's an example from usb-connection.js:

```js
// System Objects
var util = require('util');
var stream = require('stream');
var events = require('events');

var Duplex = stream.Duplex;
var Emitter = events.EventEmitter;
```


Please follow conventions—it reduces cognitive load for incoming contributors and future readers/maintainers of the code base.